tests/e2e/specs/test.js

```diff
@@ -10,6 +10,8 @@ module.exports = {
     browser
       .url(process.env.VUE_DEV_SERVER_URL)
       .waitForElementVisible('#app', 5000)
+      .click('.jumbotron button.btn')
+      .waitForElementVisible('.navbar', 5000)
```

Comment:
The e2e test finds a bug I think.
The context api is stubbed with `showCookieWall: false` but the PageComponent shows it anyways.